intelligent-service/src/quadraticcontroller.py

```diff
@@ -19,9 +23,52 @@ def quadratic():
             coeff_b = float(arg_b)
             coeff_c = float(arg_c)
         except ValueError:
-            return {"message":"At least a parameter is not a number"},406
+            return {"message": "At least a parameter is not a number"}, 406
         if coeff_a == 0:
-            return {"message":"It is linear"}, 406
+            return {"message": "It is linear"}, 406
     else:
-        return {"message":"Missing parameter"}, 406
+        return {"message": "Missing parameter"}, 406
     return quadratic_service.solve_quadratic(coeff_a, coeff_b, coeff_c), 200
+
+
+@quadratic_api.route('/qe/async/solve', methods=['POST'])
+def async_quadratic():
+    """Returns the acceptance of the async quadratic equation solver task"""
+    if not request.is_json:
+        return {"message": "Wrong format"}, 406
+    content = request.get_json()
+    print(content)
+    if len(content.get("payload")) == 3:
+        arg_a = content.get("payload")[0]
+        arg_b = content.get("payload")[1]
+        arg_c = content.get("payload")[2]
+        try:
+            coeff_a = float(arg_a)
+            coeff_b = float(arg_b)
+            coeff_c = float(arg_c)
+        except ValueError:
+            return {"message": "At least a payload element is not a number"}, 406
+        if coeff_a == 0:
+            return {"message": "It is linear"}, 406
+    else:
+        return {"message": "Wrong number of parameters"}, 406
+    if "taskId" not in content:
+        return {"message": "Missing taskId"}, 406
+    task_id = content.get("taskId")
```

Comment:
here you should write something like this

quadratic_service.solve_quadratic_async(taskId, [coeff_a, coeff_b, coeff_c])

This service will initialize the thread